documentation/staging/content/userguide/introduction/architecture.md

```diff
@@ -130,7 +128,7 @@ The diagram below shows the components inside the containers running WebLogic Se
 
 {{< img "Inside a container" "images/inside-a-container.png" >}}
 
-The Domain specifies a container image, defaulting to `container-registry.oracle.com/middleware/weblogic:12.2.1.4`. All containers running WebLogic Server use this same image. Depending on the use case, this image could contain the WebLogic Server product binaries or also include the domain directory.
+The Domain specifies a container image, defaulting to `container-registry.oracle.com/middleware/weblogic:12.2.1.4`. All containers running WebLogic Server use this same image. Depending on the use case, this image could contain the WebLogic Server product binaries or also include the domain directory. For detailed information about obtaining the operator image, see [here]({{< relref "/userguide/managing-operators/preparation#prepare-operator-image" >}}).
```

Comment:
The context seems to be more about the domain image than the operator image. It's fine to keep the operator image reference but consider also adding a reference to 
`[WebLogic Server images]({{< relref "/userguide/base-images/_index.md" >}})`

(Side note: Derek and I are in the finishing stages of putting out a major revision to the image document.)

Reply:
@tbarnes-us You're absolutely right. I changed the text and x-ref to refer to domain images.